### PR TITLE
Ux/improve right panel

### PR DESCRIPTION
**Pull request by HenriChabert**
Minor changes on design proposed by Olivier.
Before: 
Capture d’écran 2020-11-20 à 11 27 11
After: 
Capture d’écran 2020-11-20 à 11 31 47

### CODE REVIEW

**Review by clemencebic — COMMENTED** [review]
The new buttons 'back' and 'next' are great, bigger and clearer !

The colors of the 'bed' and the 'chair' categories look very similar on the screenshot, when they are displayed in the 'Categories' section, but maybe it is just because the 'chair' category is selected.

**Comment by HenriChabert** [comment]
@clemencebic You are definitely right, the new panel has colors redesigned by Olivier but the ones on bed and chair are too similar, I will find something better, thanks for the feedback

**Comment by andreybavt** [comment]
History repeats itself :) Here's a dev-in-progress UI we had last year 
image

**Comment by andreybavt** [comment]
I agree with @clemencebic about the colors, I wonder if we should actually keep the initial color preset  taken [from here](https://sashamaps.net/docs/resources/20-colors/). In the end, it seems that the guy has done some research about it, there's also a color blindness note about it:
> Approximately 95% of the general population has normal color vision and would have full access to the original palette.

File: resource/labeling-app/style.css [hunk 1]
```diff
@@ -279,11 +304,12 @@ audio {
     text-transform: capitalize;
     font-family: arial, sans-serif;
     color: var(--grey-lighten-3);
+    background-color: #F2F2F2;
```
Comment (andreybavt) [inline resource/labeling-app/style.css:307]: please use `@grey-lighten-8` instead

File: resource/labeling-app/components/category-selector.js [hunk 2]
```diff
@@ -200,10 +200,19 @@ let CategorySelector = {
                         <i v-if="status === 'SKIPPED'" class="fas fa-forward skipped"></i>
                         <h2 v-if="status === 'SKIPPED'">Sample was skipped</h2>
                     </div>
-                    <div v-else>
-                        <div v-if="status !== 'SKIPPED'" class="circle"></div>
+                    <div v-else style="width: 100%;">
                         <h2 v-if="status !== 'SKIPPED'">No labels yet</h2>
-                        <p>Select a label and draw a box around the target.</p>
+                        <p style="margin: auto;">Select a label and draw a box around the target then...</p>
```
Comment (andreybavt) [inline resource/labeling-app/components/category-selector.js:205]: Not sure I understand the phrase:
> Select a label and draw a box around the target then... Click on the categories buttons

you actually click on a category button in order to select it. 

image



I'd reformat it to be:

NO LABELS YET
Select a label by
[bubble 1](Clicking on the categories buttons) or [bubble 2](Using keyboard shortcuts)

... and draw a box around the target

Comment (andreybavt) [reply resource/labeling-app/components/category-selector.js:205]: also after omitting the "draw a box..." part this same hint can be added to the rest of webapps, not only object labeling

**Review by andreybavt — COMMENTED** [review]
are `category.png` and `shortcut.png` used somewhere?

File: resource/labeling-app/components/category-selector.js (hunk 2 above)
Comment (OMignard) [reply resource/labeling-app/components/category-selector.js:205]: I agree the wording here is weird. We could have two different versions between lal and tal going like this:

> Highlight the text to label then apply a category by…  clicking […] Using […]

> Draw a box around the target then apply a category by… clicking […] using […]

**Comment by OMignard** [comment]
> You are definitely right, the new panel has colors redesigned by Olivier but the ones on bed and chair are too similar, I will find something better, thanks for the feedback
@andreybavt @HenriChabert It seems the colors doesn't render exactly like in the rgb list i gave you. Like I try to pipet the color from your screenshot and i don't have the same rgb code. We can definitely try to solve that by changing the color back to the original for those two but i want to keep the improvment we did for the other values. Also, I'll display the color as round solid circle in the prediction to ease distinction like here:
image
And on the panel, we should align the text on the left in the button and the list to ease readability.
Thanks

File: resource/labeling-app/components/category-selector.js (hunk 2 above)
Comment (HenriChabert) [reply resource/labeling-app/components/category-selector.js]: Changed made in 0fa643f

File: resource/labeling-app/style.css (hunk 1 above)
Comment (HenriChabert) [reply resource/labeling-app/style.css]: Fixed in 44829ec

**Comment by andreybavt** [comment]
@HenriChabert 

After looking at the message again I think it's actually better to enforce selecting the category first like:

image


Because if you first draw the bbox then you'll have to manually select it to change the category. While it's still possible to do it that way I'd suggest the easier path as default to users
(I'll commit this change)

**Review by andreybavt — APPROVED** [review]

**Pull request merged**